@eikef: can you merge now?

cannot reproduce the failing check...
```
2017-10-09 15:29:20 Run command: pycodestyle --max-line-length 160 --config /dev/null --ignore E305,E402,E722,E741 lib/ansible/modules/monitoring/zabbix_host.py
2017-10-09 15:29:20 ERROR: Found 1 pep8 issue(s) which need to be resolved:
2017-10-09 15:29:20 ERROR: test/sanity/pep8/legacy-files.txt:208:1: A201 Remove "lib/ansible/modules/monitoring/zabbix_host.py" since it passes the current rule set (75%)
```

```
[schast@pc01:~/repos/github/schast-ansible.git (devel)]$ pycodestyle -v --max-line-length 160 --config /dev/null --ignore E305,E402,E722,E741 lib/ansible/modules/monitoring/zabbix_host.py
local configuration: in /data01/WORKING/repos/github/schast-ansible.git
checking lib/ansible/modules/monitoring/zabbix_host.py
[schast@pc01:~/repos/github/schast-ansible.git (devel)]$ echo $?
0
```

--- a/lib/ansible/modules/monitoring/zabbix_host.py
+++ b/lib/ansible/modules/monitoring/zabbix_host.py
@@ -223,6 +223,7 @@ import copy
 try:
     from zabbix_api import ZabbixAPI, ZabbixAPISubClass
 
+
     # Extend the ZabbixAPI
     # Since the zabbix-api python module too old (version 1.0, no higher version so far),
     # it does not support the 'hostinterface' api calls,
@@ -233,6 +234,7 @@ try:
         def __init__(self, server, timeout, user, passwd, **kwargs):
             ZabbixAPI.__init__(self, server, timeout=timeout, user=user, passwd=passwd)
             self.hostinterface = ZabbixAPISubClass(self, dict({"prefix": "hostinterface"}, **kwargs))
+
 
     HAS_ZABBIX_API = True
 except ImportError:

